drizzlepac/pars/hap_pars/user_parameters/acs/wfc/acs_wfc_quality_control_all.json

```diff
@@ -1,3 +1,26 @@
 {
-    "placeholder": "QC"
+    "ci filter": {
+      "aperture": {
+        "bthresh": 1.5,
+        "ci_lower_limit": 0.9,
+        "ci_upper_limit": 1.23
+      },
+      "segment": {
+        "bthresh": 5.0,
+        "ci_lower_limit": 0.9,
+        "ci_upper_limit": 1.23
+      }
+    },
+    "SWARM FILTER": {
```

Comment:
JSON files are case-sensitive, so upper-case SWARM FILTER could be an error unless you handle this in your code.

Reply:
Issue resolved in commit [339a927](https://github.com/spacetelescope/drizzlepac/pull/462/commits/339a92741549e37a3edc881de5891658b17e1ad7)